frontend/jupyter/src/app/pages/form/form-new/form-data-volumes/form-data-volumes.component.ts

```diff
@@ -50,6 +51,7 @@ export class FormDataVolumesComponent implements OnInit {
 
     volGroup.get('mount').setValue(`/home/jovyan/vol-${this.volsArray.length}`);
     volGroup.get('mount').markAsTouched();
+    this.newestItem = volId;
```

Comment:
When testing this locally, I get an error in the console
![image](https://github.com/StatCan/jupyter-apis/assets/84033116/743e5ef9-4351-47ed-b476-c01e727bd382)

To replicate, just click the "add existing data volume button" twice to add two new data volumes.
I think it might just be a case where this line needs to be higher in the function. But that might not be the case.

Reply:
Still present after the recent commits, but now noticing this is also happening with main when running it locally. Doesn't seem to happen on dev though. So might be an issue that only firefox returns or something. Reguardless, not an issue with this PR